Rename pulp_docker to pulp_container [wip]

ipanova opened the pull request:
[noissue]

ipanova committed:
Rename pulp_docker to pulp_container

[noissue]

pep8speaks commented:
Hello @ipanova! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:























* In the file [`pulp_container/app/tasks/synchronize.py`](https://github.com/pulp/pulp_docker/blob/c00a85d1da995c3372e04d9288c26a61362fd4d3/pulp_container/app/tasks/synchronize.py):

> [Line 46:101](https://github.com/pulp/pulp_docker/blob/c00a85d1da995c3372e04d9288c26a61362fd4d3/pulp_container/app/tasks/synchronize.py#L46): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (102 > 100 characters)





* In the file [`pulp_container/app/urls.py`](https://github.com/pulp/pulp_docker/blob/c00a85d1da995c3372e04d9288c26a61362fd4d3/pulp_container/app/urls.py):

> [Line 14:101](https://github.com/pulp/pulp_docker/blob/c00a85d1da995c3372e04d9288c26a61362fd4d3/pulp_container/app/urls.py#L14): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 100 characters)

* In the file [`pulp_container/app/viewsets.py`](https://github.com/pulp/pulp_docker/blob/c00a85d1da995c3372e04d9288c26a61362fd4d3/pulp_container/app/viewsets.py):

> [Line 369:101](https://github.com/pulp/pulp_docker/blob/c00a85d1da995c3372e04d9288c26a61362fd4d3/pulp_container/app/viewsets.py#L369): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (102 > 100 characters)

ipanova committed:
Remove content_origin and let pulp-operator auto set it.

Required PR: https://github.com/pulp/pulpcore/pull/358
Required PR: https://github.com/PulpQE/pulp-smash/pull/1227

[noissue]
